tests/std/tests/LWG2597_complex_branch_cut/test.cpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <complex>
+#include <limits>
+
+using namespace std;
+
+template <typename T>
+void check(T x, T y) {

```

Comment:
Suggestion: These could be `const`. It isn't critical, and old code is definitely inconsistent, but in new code (including test code) we try to pass values by `const` when possible. This makes modifiable value parameters stand out more.

```suggestion
void check(const T x, const T y) {
```